modules/olm-operatorgroups-copied-csvs.adoc

```diff
@@ -10,3 +10,38 @@ OLM creates copies of all active member CSVs of an Operator group in each of the
 Copied CSVs have a status reason `Copied` and are updated to match the status of their source CSV. The `olm.targetNamespaces` annotation is stripped from copied CSVs before they are created on the cluster. Omitting the target namespace selection avoids the duplication of target namespaces between tenants.
 
 Copied CSVs are deleted when their source CSV no longer exists or the Operator group that their source CSV belongs to no longer targets the namespace of the copied CSV.
+
+[NOTE]
+====
+By default, the `disableCopiedCSVs` toggle is disabled. After enabling a `disableCopiedCSVs` toggle, the OLM deletes existing copied CSVs on a cluster. When a `disableCopiedCSVs` toggle is disabled, the OLM adds copied CSVs again.
+
+* Disable the `disableCopiedCSVs` toggle:
++
+[source,yaml]
+----
+$ cat << EOF | oc create -f -
+apiVersion: operators.coreos.com/v1
+kind: OLMConfig
+metadata:
+  name: cluster
+spec:
+  features:
+    disableCopiedCSVs: false
+EOF
+----
+
+* Enable the `disableCopiedCSVs` toggle:
++
+[source,yaml]
+----
+$ cat << EOF | oc create -f -
```

Comment:
@xenolinux it should be `oc apply`, not `oc create` because OLMConfig exists by default.

Reply:
Addressed